content/en/metrics/advanced-filtering.md

```diff
@@ -77,19 +75,19 @@ Tag value prefix and suffix wildcard matching is supported:
 **Note**: Prefix and suffix wildcard matching in the same filter is not supported.
 
 
-#### Wildcard filtered query examples
+### Wildcard filtered query examples
 
 ```
 avg:system.disk.in_use{!device:/dev/loop*} by {device}
 ```
 
-{{< img src="metrics/advanced-filtering/wildcards1.mp4" alt="Example 1" video=true style="width:80%;" >}}
+{{< img src="metrics/advanced-filtering/wildcard_prefix.png" alt="Wildcard used as prefix" style="width:100%;" >}}
```

Comment:
This image doesn't match the query example. I think it needs to be swapped with `metrics/advanced-filtering/wildcard_suffix.png`

Reply:
Thanks for catching this!